packages/dashboard/src/components/app.tsx

```diff
@@ -2,73 +2,27 @@ import '@fontsource/roboto/300.css';
 import '@fontsource/roboto/400.css';
 import '@fontsource/roboto/500.css';
 import '@fontsource/roboto/700.css';
-import { createMuiTheme, ThemeProvider } from '@material-ui/core';
+import { ThemeProvider } from '@material-ui/core';
 import React from 'react';
-import { NotFoundPage } from 'react-components';
```

Comment:
Yeah, this is the easiest behaviour I found to implement. Because the admin page brings in a bunch of extra conditions like

* Is user an admin
* Does user/role exist
* Is the route a valid prefix

It makes it a lot harder to figure out when to display the not found page.